src/emulator/host/include/host/config.h

```diff
@@ -40,11 +40,11 @@ struct Config {
     bool pstv_mode = false;
     bool show_gui = false;
     bool show_game_background = true;
-    std::string pref_path;
+    std::string pref_path = {};
     bool archive_log = false;
     bool texture_cache = true;
     optional<bool> wait_for_debugger;
-    std::string background_image;
-    float background_alpha;
-    int icon_size;
+    std::string background_image = {};
+    float background_alpha = 0.000f;
```

Comment:
Set default alpha to something that people can see (in mid of not visible and totally visible, 0.5f or 129.0f? I'm not sure)

Reply:
value is already set by default on 0.000f, and this value is for full transparency, but yes, you have right, for game i recommande 0.300 for see game title correctly

Reply:
Yeah

Reply:
fixed so